ui-tests/tools/com.liferay.ide.ui.workspace/com.liferay.ide.ui.workspace.wizard/com.liferay.ide.ui.workspace.wizard.create.tomcat.70/src/com/liferay/ide/ui/workspace/tests/NewLiferayWorkspaceWizardMavenTests.java

```diff
@@ -157,6 +159,108 @@ public void createLiferayWorkspaceInitBundle() {
 		viewAction.project.closeAndDelete(project.getName());
 	}
 
+	@Test
+	public void createLiferayWorkspaceCheckPomWithDeleteModule() {
+		wizardAction.openNewLiferayWorkspaceWizard();
+
+		wizardAction.newLiferayWorkspace.prepareMaven(project.getName());
+
+		wizardAction.finish();
+
+		String projectName = "test-portlet";
+
+		wizardAction.openNewLiferayModuleWizard();
+
+		wizardAction.newModule.prepareMaven(projectName, PORTLET);
+
+		wizardAction.finish();
+
+		String modulePomXmlText = "<module>" + projectName + "</module>";
+
+		viewAction.project.openFile(project.getName(), project.getName() + "-modules (in modules)", "pom.xml");
+
+		editorAction.pomXml.switchTabPomXml();
+
+		SWTBotAssert.assertContains(modulePomXmlText, editorAction.getContent());
+
+		editorAction.close();
+
+		String[] projectNames = {project.getName(), project.getName() + "-modules (in modules)", projectName};
+
+		viewAction.project.closeAndDeleteFromDisk(projectNames);
+
+		// wait for IDE-3987 fixed
+
+		// viewAction.project.openFile(project.getName(), project.getName() + "-modules (in modules)", "pom.xml");
+
+		// editorAction.pomXml.switchTabPomXml();
+
+		// SWTBotAssert.assertDoesNotContain(modulePomXmlText, editorAction.getContent());
+
+		// editorAction.close();
+
+		String[] moduleNames = {project.getName(), project.getName() + "-modules (in modules)"};
+		String[] themeNames = {project.getName(), project.getName() + "-themes (in themes)"};
+		String[] warNames = {project.getName(), project.getName() + "-wars (in wars)"};
+
+		viewAction.project.closeAndDelete(moduleNames);
+		viewAction.project.closeAndDelete(themeNames);
+		viewAction.project.closeAndDelete(warNames);
+
+		viewAction.project.closeAndDelete(project.getName());
+	}
+
+	@Test
+	public void createLiferayWorkspaceCheckPomWithDeleteWar() {
+		wizardAction.openNewLiferayWorkspaceWizard();
+
+		wizardAction.newLiferayWorkspace.prepareMaven(project.getName());
```

Comment:
switch liferay workspace name and module name, set module name to be "project.getName()"